examples/pytorch_lightning/aim.py

```diff
@@ -0,0 +1,103 @@
+# Note: The model and training settings do not follow the reference settings
+# from the paper. The settings are chosen such that the example can easily be
+# run on a small dataset with a single GPU.
+
+import pytorch_lightning as pl
+import torch
+import torchvision
+from torch import nn
+
+from lightly.models import utils
+from lightly.models.modules import AIMPredictionHead, MaskedCausalVisionTransformer
+from lightly.transforms import AIMTransform
+
+
+class AIM(pl.LightningModule):
+    def __init__(self) -> None:
+        super().__init__()
+
+        img_size = 224
+        self.patch_size = 32
+        self.num_patches = (img_size // self.patch_size) ** 2
```

Comment:
Similar for `patch_size` and you can use `vit.patch_embed.patch_size[0]`. This way you wouldn't need to have the attribute `patch_size` for the class `AIM`